services/mirror/mirror_pull.go

```diff
@@ -215,13 +216,21 @@ func runSync(ctx context.Context, m *repo_model.Mirror) ([]*mirrorSyncResult, bo
 		return nil, false
 	}
 
+	envs := []string{}
```

Comment:
```suggestion
	envs :=os.Environ()
```

Reply:
Agree to keep the old behavior. Users may have set their own `https_proxy` or other env var for the whole Gitea process.

Otherwise many env var behaviors should be refactored and be marked as breaking.

Reply:
> Agree to keep the old behavior. Users may have set their own https_proxy or other env var for the whole Gitea process.

Great that the usage of `os.Environ()` was documented, I honestly thought that this was some leftover from gogs code.